tests/unit/specs/service/auth.spec.js

```diff
@@ -152,4 +152,40 @@ describe('auth service', () => {
       });
     });
   });
+
+  describe('authWithGoogle', () => {
+    const url = `${identityBaseUrl}/auth/google`;
+
+    it('should return 200 OK', async () => {
```

Comment:
what about tests with fail result?

Reply:
Agree with @dudiq, it would be great to handle unsuccessful cases in tests.

Reply:
@lamartire @kazurus  done.